certbot/certbot/_internal/main.py

```diff
@@ -534,12 +534,21 @@ def _report_next_steps(config: configuration.NamespaceConfig, installer_err: Opt
 
     # If the installation or enhancement raised an error, show advice on trying again
     if installer_err:
-        steps.append(
-            "The certificate was saved, but could not be installed (installer: "
-            f"{config.installer}). After fixing the error shown below, try installing it again "
-            f"by running:\n  {cli.cli_command} install --cert-name "
-            f"{_cert_name_from_config_or_lineage(config, lineage)}"
-        )
+        # Special case where either --nginx or --apache were used, causing us to
+        # run the "installer" (i.e. reloading the nginx/apache config)
+        if config.verb == 'certonly':
+            steps.append(
+                "The certificate was saved, but was not successfully loaded by the installer"
+                f"({config.installer}) due to the installer failing to reload."
+                f"After fixing the error shown below, try reloading {config.installer} manually."
```

Comment:
I think the biggest difference between the two is if we're reporting overall success or failure here. I haven't actually traced through the code to be sure, but I think this should be the last thing certbot tries to do before closing anyway, so unless something changes in the future, it may just be a question of UX (we should probably double check this).

If it's just a question of UX, I agree with leaning towards erroring out but could be persuaded either way. On the other hand, we wouldn't want to cause undue worry; they still did get the cert successfully after all. I also think it's unlikely to be a common case, since we're just refreshing something that had to already have been working to get this far.

Reply:
ah i hadn't seen that! +1 to certbot erroring out in this case, since a warning w/ successful return code seems easy to miss.